app/models/admin/police/builder.js

```diff
@@ -0,0 +1,39 @@
+const { routes, breadcrumbs } = require('../../../constants/admin')
+const ViewModel = require('../success')
+
+/**
+ *
+ * @param policeForce
+ * @return {*}
+ */
+const PoliceForceAddedViewModel = (policeForce) => {
+  return new ViewModel({
+    breadcrumbs,
+    titleHtml: `You added<br /> ${policeForce}`,
+    bodyContent: [`${policeForce} is available in the Index.`],
+    bottomLink: {
+      link: routes.police.get,
+      label: 'Add or remove a police force'
+    }
+  })
+}
+
+const PoliceForceRemovedViewModel = (policeForce) => {
+  return new ViewModel({
+    breadcrumbs,
+    titleHtml: `You removed<br /> ${policeForce}`,
+    bodyContent: [
```

Comment:
Nothing really to do with this review but I don;t like the wording of 'and will not be available for new applications' since that's not quite right. Would not be available if changing existing applications or creating new applications. Not sure if we should raise it with Nick/Claire or whether it's clear enough as is

Reply:
Are you able to bring this one up?  We could sort as a bugfix if need be.